kube/harbor-scanner-trivy.yaml

```diff
@@ -38,28 +38,6 @@ spec:
       labels:
         app: harbor-scanner-trivy
     spec:
-      initContainers:
-        - name: init
-          image: aquasec/harbor-scanner-trivy:0.1.0-rc1
-          imagePullPolicy: IfNotPresent
-          command:
-            - "trivy"
-            - "--debug"
```

Comment:
You can use `--download-db-only` option in initContainers. But if Harbor accepts a few seconds on the first run, `initContainers` is unnecessary.

Reply:
IMHO it's acceptable to have the first scan a bit slower for simplifying K8S deployment descriptors.

Reply:
I see